yadm

```diff
@@ -1819,6 +1833,13 @@ function auto_bootstrap() {
 
 }
 
+# ****** Helper Functions ******
+
+function join_string {
+    local IFS="$1"
+    echo "${*:2}"
```

Comment:
This may fail on files that start with a `-`. Consider using `printf`  instead. See https://askubuntu.com/a/467756

Reply:
Or do you intend to call the `echo` function defined below?

Reply:
I'm really learning something here! :smile: Thanks!!!

I got rid of all other `echo` instances, too, and changed them to `printf`. Just to get used to it...